app/controllers/application_controller.rb

```diff
@@ -85,7 +85,7 @@ def prompt_to_enter_otp
   end
 
   def track_get_requests
-    return unless request.get?
+    return unless request.get? && request.fullpath != '/active'
```

Comment:
Good point, Peter. Are there other requests we want to ignore too? I imagine our future status endpoints won't need to be included as analytics events as well.